uber/models.py

```diff
@@ -1252,6 +1263,9 @@ class Attendee(MagModel, TakesPaymentMixin):
     ec_phone      = Column(UnicodeText)
     cellphone     = Column(UnicodeText)
 
+    # Represents a request for hotel booking info during preregistration
+    requested_hotel_info = Column(Boolean, default=False)
```

Comment:
I actually went back and forth on whether this should default to `True` or `False`. I ended up making it false because the window of time where this will be relevant is tiny.

So for everyone in the database where this value is `True`, it's because they really did want the email, or we explicitly decided they should receive it.

For the preregistration page, the default is definitely `True`. So if an attendee just clicks through during preregistration without reading anything they will still receive the email.